.github/workflows/lint.yml

```diff
@@ -13,4 +13,6 @@ jobs:
     steps:
       - uses: actions/checkout@v3
       - uses: actions/setup-python@v2
+        with:
+          python-version: "3.11"
```

Comment:
I think this one shouldn't be pinpointed. The problem is in pre-commit.

Reply:
> I think this one shouldn't be pinpointed. The problem is in pre-commit.

I think otherwise. Github action should not behave differently when suddenly other python version is used.

Reply:
I think we must pin the python version for pre-commit, just like we pin it for running tests.
Otherwise we hit this kind of issue every time GitHub upgrades its default python.

My suggestion would be to use the same python version as for the tests.

Reply:
Not sure what you want to say here. Can you do a suggestion if still on time?